### ISSUE TITLE
Clippy installation fails nightly

### ISSUE THREAD

**Issue by elpiel** [sagiegurari/cargo-make#290]
**Describe the bug**
After I've updated my nightly toolchain to the latest update, my clippy had to be rebuild, so cargo-make tried to update it by failed.

**To Reproduce**
1. Update to latest nightly (or a toolchain that will trigger rebuild/reinstall of clippy)
2. Run `cargo-make install-clippy-any`

**Error Stack**
```console
[cargo-make] INFO - Execute Command: "cargo" "install" "--force" "--git" "https://github.com/rust-lang/rust-clippy/"
    Updating git repository `https://github.com/rust-lang/rust-clippy/`
error: multiple packages with binaries found: clippy, clippy_dev, clippy_dummy, clippy_workspace_tests
```

Changing the install arguments fixes the problem:
`install_crate_args = ["--git", "https://github.com/rust-lang/rust-clippy/", "clippy"]`

I wanted to fire the bug report and confirm this is the case, before making a PR.

Regards

**Comment by sagiegurari** [comment]
always happy to get a PR :)

**Comment by elpiel** [comment]
Just wanted to check if I am the only one and that there is indeed a problem there.
Look out for my PR 😋

**Comment by sagiegurari** [comment]
published in 0.22.2 release
